enso/featurize/transformer_features.py

```diff
@@ -0,0 +1,90 @@
+import sys
+import tensorflow as tf
+import os
+import os.path
+import transformer
+
+# TODO: change this to a dependency on the "transformer"
+# package when implementation / saved weights are stable
+import_path = os.path.abspath(
+    os.path.join(
+        os.path.dirname(os.path.dirname(transformer.__file__)),
+        "data/models/SNLI"
+    )
+)
+sys.path.append(import_path)
+
+from v1.model import Transformer
```

Comment:
Should probably add this to Enso. Maybe we have a directory like the ML directory in indicoAPI that is for source files like these. Would be very surprised if this is the last time this happens.

Potentially we also make a privvy entry for the weight file for maximum reproducability.

Reply:
I'm not sure what the right approach is here.  Adding all the experimental models into enso directly seems... undesirable.  Among other things it locks us into specific library versions between featurizer projects.

Reply:
I can remove this file from the PR if you like.

Reply:
hmmm. Yea, that's a good point. Not really sure what the best way of handling this is. Once it's in indico API it's easy enough. I think I'd rather include experiments than remove things alltogether, but at the same time it should be a pretty limited to make it not awful